assets/overlays/openstack-cinder/patches/controller_add_driver.yaml

```diff
@@ -0,0 +1,111 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  annotations:
+    config.openshift.io/inject-proxy: csi-driver
+    config.openshift.io/inject-proxy-cabundle: csi-driver
+spec:
+  template:
+    metadata:
+      annotations:
+        # FIXME(stephenfin): This was not set previously for openstack-cinder-csi-driver-operator.
+        # but is set in assets/base/controller.yaml. Is settting it to the empty string enough, or
+        # should we remove it from the base and move it into the drivers that need it?
+        cluster-autoscaler.kubernetes.io/safe-to-evict-local-volumes: ""
+        # FIXME(stephenfin): This was not set previously for openstack-cinder-csi-driver-operator.
+        # What is the effect/necessity of setting it now?
+        openshift.io/required-scc: privileged
```

Comment:
I can't do that (I did try) because we use host networking for the controller. Not sure why the controller needs host networking but I was copying what was done previously

Reply:
Yep, not sure. It [was there since the beginning](https://github.com/openshift/openstack-cinder-csi-driver-operator/commit/a6c64477a530b7b132a0a61d75237606b7c4e045#diff-529b166853e98d1126b70e972faa93f1355cb4283af58abf08d9d96509d00cbdR17).